common/middleware/ensure-authenticated.js

```diff
@@ -9,13 +9,33 @@ function _isExpired(authExpiry) {
 module.exports = function ensureAuthenticated({
   provider,
   whitelist = [],
+  expiryMargin = 5 * 60,
 } = {}) {
   return (req, res, next) => {
-    if (whitelist.includes(req.url) || !_isExpired(req.session.authExpiry)) {
+    let authExpiry = req.session.authExpiry
+    if (req.method === 'GET') {
+      authExpiry -= expiryMargin
+    }
+    if (whitelist.includes(req.url) || !_isExpired(authExpiry)) {
       return next()
     }
 
     req.session.originalRequestUrl = req.originalUrl
+    if (req.method === 'POST') {
+      let errorKey
+      const contentType = req.header('content-type')
+      if (contentType.startsWith('multipart/form-data')) {
```

Comment:
Minor comment but I find it's a bit easier to read code when there is a bit of breathing room between blocks. Like a new line before/after an `if/else` block or after a block of variables being defined.

Reply:
Can we not add that to our prettifier/eslint rules?

Reply:
(I've stuffed some new lines in)

Reply:
Possibly, but not sure if it can be that clever...maybe one to look into